Release article: 4.77 (#36148)
fleetdm/fleet: articles/fleet-4.77.0.md

- Reduced Android device reconciler frequency to 1 hour.
- Reduced Android API usage by listing devices instead of getting and checking Android Enterprise disconnects hourly.
- Set the order of software installed during the setup experience to alphanumeric.
- Updated Go to 1.25.3.
- Fixed a layout issue on the script batch details page.
- Fixed installer for Cisco Secure Client not showing as installed in inventory/library due to using the wrong bundle identifier. This application should show up correctly now in the software inventory.
- Fixed errors when trying to run the `apple_mdm_iphone_ipad_refetcher` cron job.
- Fixed bug that prevented users from editing custom EST certificates URLs.
- Fixed incorrect UI placeholder element by replacing it with it's actual value.
- Fixed issue where vulnerabilities would occasionally show as missing.

## Ready to upgrade?

Visit our [Upgrade guide](https://fleetdm.com/docs/deploying/upgrading-fleet) in the Fleet docs to update to Fleet 4.77.0.

<meta name="category" value="releases">
<meta name="authorFullName" value="Noah Talerman">
<meta name="authorGitHubUsername" value="noahtalerman">
<meta name="publishedOn" value="2025-12-04">

> <meta name="articleTitle" value="Fleet 4.77.0 | iOS/iPadOS self-service and enterprise packages, edit IdP username, and more...">

Reviewer: ```suggestion
<meta name="articleTitle" value="Fleet 4.77.0 | Deploy enterprise packages for iOS/iPadOS, edit IdP username, and more...">
```

# Fleet 4.77.0 | iOS/iPadOS self-service and enterprise packages, edit IdP username, and more...

<div purpose="embedded-content">
   <iframe src="https://www.youtube.com/embed/2oLyaV7rIXM?si=FgWAi9K8KhEXWd_f" frameborder="0" allowfullscreen></iframe>
</div>

Fleet 4.77.0 is now available. See the complete [changelog](https://github.com/fleetdm/fleet/releases/tag/fleet-v4.77.0) or read on for highlights. For upgrade instructions, visit the [upgrade guide](https://fleetdm.com/docs/deploying/upgrading-fleet) in the Fleet docs.

## Highlights

> - iOS/iPadOS self-service

Reviewer: ```suggestion
```

This was pushed to 4.78.0.

### Activity for deleted hosts

Deleted host events are now included in the activity feed. If a host is removed, you’ll still have a record for audits and historical tracking.

### More Fleet-maintained apps

Fleet added [Fleet-maintained apps](https://fleetdm.com/guides/fleet-maintained-apps) for macOS (Claude, ChatGPT, Outlook, Webex, Spotify) and Windows (Slack, Zoom, Firefox), plus many more apps. See all Fleet-maintained apps in the [software catalog](https://fleetdm.com/software-catalog).

## Changes

TODO

## Ready to upgrade?

Visit our [Upgrade guide](https://fleetdm.com/docs/deploying/upgrading-fleet) in the Fleet docs to update to Fleet 4.77.0.

<meta name="category" value="releases">
<meta name="authorFullName" value="Noah Talerman">
<meta name="authorGitHubUsername" value="noahtalerman">

> <meta name="publishedOn" value="2025-11-24">

Reviewer: ```suggestion
<meta name="publishedOn" value="2025-12-04">
```

> <meta name="articleTitle" value="Fleet 4.77.0 | iOS/iPadOS self-service and enterprise packages, edit IdP username, and more...">

Reviewer: Need to change description to not include iOS/iPadOS self-service.
